README.md

```diff
@@ -142,6 +142,43 @@ The following platforms are currently not supported with the gProfiler executabl
 See [gprofiler.yaml](deploy/k8s/gprofiler.yaml) for a basic template of a DaemonSet running gProfiler.
 Make sure to insert the `GPROFILER_TOKEN` and `GPROFILER_SERVICE` variables in the appropriate location!
 
+## Running as an ECS (Elastic Container Service) Daemon service
+#### Creating the ECS Task Definition
+- Go to ECS, and [create a new task definition](https://console.aws.amazon.com/ecs/home?region=us-east-1#/taskDefinitions/create)
+- Choose EC2, and click `Next Step`
+- Scroll to the bottom of the page, and click `Configure via JSON` \
+![Configure via JSON button](images/configure_via_json.jpg)
+- Replace the JSON contents with the contents of the [gprofiler_task_definition.json](deploy/ecs/gprofiler_task_definition.json) file and **Make sure you change the following values**:
+  - Replace `<CLIENT TOKEN>` in the command line with your token you got from the [gProfiler Performance Studio](https://profiler.granulate.io/) site
```

Comment:
Nitpick: these could be consistent (the placeholder strings) across k8s, docker-compose, ECS etc.